lib/profileAgents.js

```diff
@@ -602,6 +609,104 @@ async function _updateMeterController({meterId, controller}) {
   }
 }
 
+async function _getProfileAgentRecords({accountId, includeSecrets}) {
+  const query = {'profileAgent.account': accountId};
+  const projection = {_id: 0};
+  const collection = getCollection('profile-profileAgent');
+  let records = await collection.find(query, {projection}).toArray();
+  const zcapRefreshed = await _refreshProfileAgentZcaps({
```

Comment:
All the code from here down to the return should be moved back to `getAll`.

Reply:
Moved https://github.com/digitalbazaar/bedrock-profile/pull/99/files#diff-23bb1aeaf4f828a3d7c81d2f74e76cbac5f577e2900ee22e1ac867cccae2ac2dR175-R180